src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -283,9 +286,19 @@ const CoverImageEditModal = ({
                 <ButtonV2
                   variant="danger"
                   onClick={handleDelete}
-                  disabled={isUploading}
+                  disabled={isDeleting}
                 >
-                  {t("delete")}
+                  {isDeleting ? (
+                    <CareIcon
+                      icon="l-spinner"
+                      className="animate-spin text-lg"
+                    />
+                  ) : (
+                    <CareIcon icon="l-save" className="text-lg" />
+                  )}
+                  <span>
+                    {isDeleting ? `${t("Deleting")}...` : `${t("delete")}`}
```

Comment:
why are we using capital key `Deleting`? 🤔

Reply:
will change it